volume-providers/vpc/vpcclient/models/constants.go

```diff
@@ -25,4 +25,10 @@ const (
 
 	// GTypeClassicDevicePrefix ...
 	GTypeClassicDevicePrefix = "/dev/"
+
+	// GTypeG2 ...
+	GTypeG2 = "g2"
+
+	// GTypeG2DevicePrefix ...
+	GTypeG2DevicePrefix = "/dev/disk/by-id/virtio-"
```

Comment:
I think prefix is `/dev/disk/virtio-`, pl double check

Reply:
verified ok.
$ /Users/shaikh/runon.sh 10.240.0.13 bash
root@dev-bm9de4g00fq8k9r85ub0-shaikhvpcng-default-00000105:/# ls -l /dev/disk/by-id/
total 0
lrwxrwxrwx 1 root root 9 Oct 21 11:43 virtio-0717_1a6d88bc-2eb8-4 -> ../../vde
lrwxrwxrwx 1 root root 9 Oct 1 05:18 virtio-0717_2125eb2a-e14f-4 -> ../../vda
lrwxrwxrwx 1 root root 10 Oct 1 05:18 virtio-0717_2125eb2a-e14f-4-part1 -> ../../vda1
lrwxrwxrwx 1 root root 10 Oct 1 05:18 virtio-0717_2125eb2a-e14f-4-part2 -> ../../vda2
lrwxrwxrwx 1 root root 9 Oct 21 11:41 virtio-0717_53148584-fa91-4 -> ../../vdd